src/wire/ipv4.rs

```diff
@@ -110,8 +110,8 @@ impl fmt::Display for Address {
 /// subnet masking prefix length.
 #[derive(Debug, Hash, PartialEq, Eq, PartialOrd, Ord, Clone, Copy, Default)]
 pub struct Cidr {
-    address:    Address,
-    prefix_len: u8,
+    pub(crate) address:    Address,
+    pub(crate) prefix_len: u8,
```

Comment:
Why is this necessary? There are accessors and constructors for these...

Reply:
I need it in order to write this:

```
const CIDR_1: Ipv4Cidr = Ipv4Cidr {      
    address: Ipv4Address([192, 0, 2, 0]),
    prefix_len: 24,                      
};                                       
```

Using a constructor is forbidden in const expressions.

Reply:
Can you make it a function instead? We can make `new` `const fn` when they land. I feel like making a change this big just for tests isn't worth it.

Reply:
Sure. Done.